src/diy/imu-comparison.md

```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
+
+
+|Reset time |Cost |Availability|Build quality|
+|:---------:|:---:|:----------:|:-----------:|
+|45-60 min|~$6 |Sufficient  |Great           |
+
+Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-half-o"></i>
+
+|Pros          |Cons                                                                     |
+|--------------|-------------------------------------------------------------------------|
+|Cheap         |Very Experimental                                                        |
+|Reliable      |Only available in module form from a single source                       |
+|Smooth        |                                                                         |
+
+
+---
+## LSM6DSO
+The LSM6DSO is a good IMU for DIY SlimeVR.
+it has been shown to be slightly better than the BMI270 at a slightly higher cost.
+
+|Reset time |Cost |Availability|Build quality|
+|:---------:|:---:|:----------:|:-----------:|
+|25-35 min|~$3.5 |Sufficient  |Great         |
+
+Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>
+
+|Pros          |Cons                                                                     |
+|--------------|-------------------------------------------------------------------------|
+|Cheap         |Experimental                                                             |
+|Reliable      |Only available in module form from a single source                       |
+|Smooth        |Poor Sensitivity                                                         |
+
 ---
 ## BMI270
-The BMI270 is a relatively new and <b>experimental</b> IMU for DIY SlimeVR.
+The BMI270 is a good IMU for DIY SlimeVR.
 It seems to perform significantly better than the BMI160 while still being affordable.
 
 |Reset time |Cost  |Availability|Build quality|
 |:---------:|:----:|:----------:|:-----------:|
-|20 - 30min |~$2.7 |Sufficient  |Great        |
+|20 - 30min |~$2.5 |Sufficient  |Great        |
 
 Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>
 
 |Pros                   |Cons                                                            |
 |-----------------------|----------------------------------------------------------------|
-|Cheap                  | [Requires manual calibration the first time](#imu-calibration) |
-|Reliable               | Only available in module form from a single source             |
-|Smooth                 | Experimental, still in testing                                 |
-|Single time calibration| Performance data not taken from survey                         |
+|Cheap                  | Only available in module form from a single source             |
+|Reliable               | Experimental                                                   | 
```

Comment:
I don't know if BMI270 should still be considered experimental, after an entire year of being available.